src/unix/linux_like/linux/gnu/b32/sparc/mod.rs

```diff
@@ -508,6 +508,10 @@ pub const TCGETS: ::c_ulong = 0x40245408;
 pub const TCSETS: ::c_ulong = 0x80245409;
 pub const TCSETSW: ::c_ulong = 0x8024540a;
 pub const TCSETSF: ::c_ulong = 0x8024540b;
+pub const TCGETS2: ::c_ulong = 0x402c542a;
+pub const TCSETS2: ::c_ulong = 0x802c542b;
+pub const TCSETSW2: ::c_ulong = 0x802c542c;
+pub const TCSETSF2: ::c_ulong = 0x802c542d;
```

Comment:
Sparc uses a different definition than other platforms:
```c
// sparc
#define TCGETS2		_IOR('T', 12, struct termios2)
#define TCSETS2		_IOW('T', 13, struct termios2)
#define TCSETSW2	_IOW('T', 14, struct termios2)
#define TCSETSF2	_IOW('T', 15, struct termios2)

// everyone else
#define TCGETS2		_IOR('T', 42, struct termios2)
#define TCSETS2		_IOW('T', 43, struct termios2)
#define TCSETSW2	_IOW('T', 44, struct termios2)
#define TCSETSF2	_IOW('T', 45, struct termios2)
```

Reply:
Good catch. Updated. I think I had the direction bits for `mips` wrong too :(

Updated those now too, so it's `0x40` for read and `0x80` for write.